### PR TITLE
feat(blog): add community tools blogpost

### PR DESCRIPTION
**Pull request by nsarrazin**
Blog post that explains how to create & use tools in HuggingChat to go along the release.

### Pre-release checklist
- [x] Add link to feedback thread when it's up on the space's discussion page

### CODE REVIEW

**Review by SocialNinjaMaster — APPROVED** [review]
Looks Good!😻

**Review by SocialNinjaMaster — APPROVED** [review]

File: community-tools.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+---
+title: "Introducing Community Tools on HuggingChat" 
+thumbnail: /blog/assets/community-tools/thumbnail.png
+authors:
+- user: nsarrazin
+---
+
+Today we’re releasing our latest feature on HuggingChat: community tools! This lets you turn any Space that you love on HuggingFace into a tool that can be used by models directly from HuggingChat.
```
Comment (gary149) [inline community-tools.md:8]:
```suggestion
Today we’re releasing our latest feature on HuggingChat: Community Tools! This lets you turn any Space that you love on HuggingFace into a tool that can be used by models directly from HuggingChat. 
```

**Review by gary149 — APPROVED** [review]

File: community-tools.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+With this feature, we’re also expanding the modalities available in HuggingChat. You can now use community tools to understand images, generate videos, or answer with a text-to-speech model. The possibilities are endless and anyone can create tools using Spaces on Hugging Face! Explore existing tools [here](https://huggingface.co/chat/tools).
+
+In this post we’re going to look at a few use cases for creating community tools:
+
+1. [Turning a community space into a tool](#turning-a-community-space-into-a-tool)
+2. [Creating a custom tool yourself](#creating-a-custom-tool-yourself)
+3. [Enhance your assistants with community tools](#enhance-your-assistants-with-community-tools)
+4. [Create a RAG tool on your own documents](#create-a-rag-tool-on-your-own-documents)
+
+## Turning a community space into a tool
```
Comment (julien-c) [inline community-tools.md:19]:
```suggestion
## Turning a community Space into a tool
```

File: community-tools.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+- user: nsarrazin
+---
+
+Today we’re releasing our latest feature on HuggingChat: Community Tools! This lets you turn any Space that you love on HuggingFace into a tool that can be used by models directly from HuggingChat. 
+
+With this feature, we’re also expanding the modalities available in HuggingChat. You can now use community tools to understand images, generate videos, or answer with a text-to-speech model. The possibilities are endless and anyone can create tools using Spaces on Hugging Face! Explore existing tools [here](https://huggingface.co/chat/tools).
+
+In this post we’re going to look at a few use cases for creating community tools:
+
+1. [Turning a community space into a tool](#turning-a-community-space-into-a-tool)
```
Comment (julien-c) [inline community-tools.md:14]:
```suggestion
1. [Turning a community Space into a tool](#turning-a-community-space-into-a-tool)
```

File: community-tools.md (hunk 2 above)
Comment (julien-c) [reply community-tools.md:19]: and in other places too

Comment (nsarrazin) [reply community-tools.md]: done with [c3d16a2](https://github.com/huggingface/blog/pull/2342/commits/c3d16a225199723841e4fca37a1f3b632e398c11)

**Pull request merged**